config/persistence.php

```diff
@@ -17,5 +18,14 @@
                 tagged_iterator('.foundry.persistence.database_resetter'),
                 tagged_iterator('.foundry.persistence.schema_resetter'),
             ])
+
+        ->set('.zenstruck_foundry.story.load_story-command', LoadStoryCommand::class)
+            ->arg('$databaseResetters', tagged_iterator('.foundry.persistence.database_resetter'))
+            ->arg('$kernel', service('kernel'))
+            ->tag('console.command', [
+                'command' => 'foundry:load-stories',
+                'aliases' => ['foundry:load-fixtures'],
```

Comment:
```suggestion
                'aliases' => ['foundry:load-fixtures', 'foundry:load-story'],
```

I'd like both `foundry:load-story` and `foundry:load-fixture` to work (`foundry:load-fixture` already works).